src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -3348,7 +3360,11 @@ void MusicXMLParserDirection::wedge(const QString& type, const int number,
         if (niente == "yes") {
             h->setHairpinCircledTip(true);
         }
-        starts.append(MusicXmlSpannerDesc(h, ElementType::HAIRPIN, number));
```

Comment:
Sure this line can get dropped?

Reply:
sorry …

Reply:
And I'm not sure whether it needs before or after the color setting

Reply:
Or whether it matter at all